### DISCUSSION TITLE
Welcome to Contribute-To-This-Project Discussions!

### DISCUSSION THREAD

**Discussion by Syknapse** [Syknapse/Contribute-To-This-Project#903] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by lax0248917** [comment]
Hello @Syknapse and fellow community members. My name is Zach and I am a software administrator/developer for a large electrical distribution company in Michigan, USA specializing in help desk ticketing and collaboration software. Spend most of my day customizing software and building custom apps in Javascript.

I am one of the Maintainers for the project. I am often doing code reviews and doing my best to help new open source contributors feel welcome. Look forward to seeing this community grow!!!!! 🚀

**Comment by moonbe77** [comment]
Hi, I'm *Berny* Frontend dev,
I am a contributor and maintainer of this project.
Happy to help anyone with their first contribution, please do not hesitate to ask anything you like.
You could also make questions in Spanish if it is more comfortable for you. Spanish is my mother tongue.

```See you in the next PR```

**Comment by 9662e103-129a** [comment]
Howdy there,
I am 324hz, a fullstack developer, and I am also a maintainer & contributer to this project.

I just started getting into AI development with the Nvidia Jetson modules and currently I am working on integrating that tech with Arduino and maybe some day with my website!

See you later!

324hz
win21H2

**Comment by confused-Techie** [comment]
Hello all, I'm confused-techie, preferred backend dev and maintainer. Appreciating the reminder here to be more active as other projects have become a time suck. Currently implementing a Backend, from scratch, for Atom.io with Atom-Community due to the Atom sunset. 


Also I saw the talk about starting a Discord, and that could be pretty cool, but I think for it to have any necessity we would have to expand the reach of this project. But on that note, @Syknapse have you ever given any thought to if we'd be able to help first time developers/users of Github in any other way? Maybe experience with created issues somehow? Or any other aspects of the platform?

**Reply by Syknapse** [reply]
To be honest I would love for this project to do as much as possible to help first time contributors and beginners. It's a really gratifying feeling to help people take those first crucial steps. 

The only issue I have is time. I hardly have time to keep up here as is. Which is why I opened the project up for maintainers. And I would encourage and support any idea that improves this project. As long as someone is willing to take ownership of the idea and see it to fruition I'm definitely on board.

**Reply by confused-Techie** [reply]
Thats very understandable, and about where I am too. But if I find the time, I'll see if I can put something else together, and maybe we could link them back to each other, or even list them as a sort of roadmap course for starters. 

But thanks for the endorsement!

**Reply by 9662e103-129a** [reply]
Howdy confused-Techie,
That sounds good. It could be a thing where there are a few people dedicated to working on the README files creating roadmaps etc..

324hz
win21H2

**Comment by ghost** [comment]
Hi everyone!

**Reply by danieljuliusstein** [reply]
hello

**Comment by khaledYS** [comment]
> ## 👋 Welcome!
> We’re using Discussions as a place to connect with other members of our community. We hope that you:
> 
> * Ask questions you’re wondering about.
> * Share ideas.
> * Engage with other community members.
> * Welcome others and are open-minded. Remember that this is a community we
>   build together 💪.
> 
> To get started, comment below with an introduction of yourself and tell us about what you do with this community.

Hi guys, I was wondering if I could contribute to add pagination to show posts in much lighter way. First time I opened the website, it started glitching a lot because of how big is the website. Just An Idia and I wanted to share it.

**Reply by Syknapse** [reply]
This could be a very good idea to make the page load quicker. If you have a good idea how to implement it we would love to get that pull request. Go for it
